Core/src/org/sleuthkit/autopsy/casemodule/UnpackagePortableCaseProgressDialog.java

```diff
@@ -261,6 +292,33 @@ private File locate7ZipExecutable() {
 
             return exeFile;
         }
+
+        /**
+         * Check to see if the case has been unpacked before
+         */
+        private String hasCaseBeenUnpackedBefore(String packedCaseName) {
```

Comment:
Can this be renamed to something like getPreviouslyUnpackedCaseFileName? I expected it to return a boolean given its name.

Reply:
Renamed.